packages/accounts/src/account.ts

```diff
@@ -455,6 +469,85 @@ export class Account {
         });
     }
 
+    /**
+     * Compose and sign a SignedDelegate action to be executed in a transaction on behalf of this Account instance
+     *
+     * @param actions Actions to be included in the meta transaction
+     * @param blockHeightTtl Number of blocks past the current block height for which the SignedDelegate action may be included in a meta transaction
+     * @param receiverId Receiver account of the meta transaction
+     */
+    async signedDelegate({
+        actions,
+        blockHeightTtl = DEFAULT_META_TRANSACTION_BLOCK_HEIGHT_TTL,
+        receiverId,
+    }: SignedDelegateOptions): Promise<SignedDelegate> {
+        const { provider, signer } = this.connection;
+        const { header } = await provider.block({ finality: 'final' });
+        const { publicKey } = await this.findAccessKey(null, null);
+
+        const delegateAction = buildDelegateAction({
+            actions,
+            maxBlockHeight: new BN(header.height).add(new BN(blockHeightTtl)),
+            nonce: await this.getSigningNonce(),
+            publicKey,
+            receiverId,
+            senderId: this.accountId,
+        });
+
+        const { signedDelegateAction } = await signDelegateAction({
+            delegateAction,
+            signer: {
+                sign: async (message) => {
+                    const { signature } = await signer.signMessage(
+                        message,
+                        delegateAction.senderId,
+                        this.connection.networkId
+                    );
+
+                    return signature;
+                },
+            }
+        });
+
+        return signedDelegateAction;
+    }
+
+    /**
+     * Compose, sign, and submit (to relayer) a set of actions to be executed in a meta transaction
+     *
+     * @param actions Actions to be included in the meta transaction
+     * @param blockHeightTtl Number of blocks past the current block height for which the SignedDelegate action may be included in a meta transaction
+     * @param receiverId Receiver account of the meta transaction
+     * @param relayerUrl Relayer URL endpoint responsible for building and executing meta transactions
+     * @param signedDelegate SignedDelegate instance containing the information and verification required for relayer execution of meta transactions
+     */
+    async signAndSendSignedDelegate({
+        actions,
+        blockHeightTtl,
+        receiverId,
+        relayerUrl,
+    }: { relayerUrl: string } & SignedDelegateOptions): Promise<FinalExecutionStatus> {
+        const signedDelegate = await this.signedDelegate({ actions, blockHeightTtl, receiverId });
+        return fetchJson(relayerUrl, JSON.stringify(Array.from(encodeSignedDelegate(signedDelegate))));
```

Comment:
Current implementation of relayers requires the JSON body be a serialized byte array (i.e. not wrapped in an object).

NEP-366 describes an interface for this, so it's possible the implementation will change:
```rs
struct DelegateActionMessage {
    signed_delegate_action: SignedDelegateAction
}
```

> - User forms `DelegateActionMessage` with the `SignedDelegateAction`.
> - User sends `DelegateActionMessage` data to the relayer.